lib/nexpose/common.rb

```diff
@@ -301,6 +301,47 @@ module Type
     end
   end
 
+  # Configuration structure for blackouts.
+  class Blackout < APIObject
```

Comment:
Let's move this class into it's own file. There's no reason to have this in the common.rb. 
We can refactor out the other non-common classes in another pull request.